pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -277,63 +283,47 @@ func ReadLogs(path, containerID string, opts *LogOptions, runtimeService interna
 	// Do not create watcher here because it is not needed if `Follow` is false.
 	var watcher *fsnotify.Watcher
 	var parse parseFunc
+	var stop bool
 	writer := newLogWriter(stdout, stderr, opts)
 	msg := &logMessage{}
 	for {
```

Comment:
This is a cleanup for https://github.com/kubernetes/kubernetes/pull/55631#issuecomment-344743772.